root/etc/s6-overlay/s6-rc.d/adsbexchange-feed/run

```diff
@@ -0,0 +1,22 @@
+#!/command/with-contenv bash
+
+if [ "$SERVICE_ENABLE_ADSBEXCHANGE" != "false" ]; then
+    set -eo pipefail
+
+    mkdir -p /run/adsbexchange-feed
```

Comment:
Sorry, I forgot to remove this line, better to create the folder in Dockerfile.
I wanted to do it but forgot. 😅

Reply:
/run should be mounted as tmpfs to avoid useless disk writes.

To make that possible it's necessary to create subfolder in the services.

Reply:
Sure it can, you can add it to the readme for compose and docker run.

Reply:
No i mean the entire /run folder, not /run/adsbexchange-feed.

Reply:
But we can create the folder in conf service, it's a setup command.